templates/azure/confidential-computing.html

```diff
@@ -0,0 +1,181 @@
+{% extends "azure/base_azure.html" %}
+
+{% block title %}Confidential computing on Microsoft Azure{% endblock %}
+
+{% block meta_description %}
+  Protect your security sensitive workloads on Microsoft Azure with Ubuntu confidential VMs
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1sVaQQlm6LHyRf-Q3VSVTmqGRi0SV7gwBVczCTPWq3CE/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero p-suru">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Confidential computing with Ubuntu on Microsoft Azure</h1>
+      </div>
+      <div class="col">
+        <p>Secure your data, code and AI models at run- time with Ubuntu confidential VMs on Microsoft Azure.</p>
+        <hr class="p-rule--muted" />
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=Canonical%20Ubuntu%20confidential%20VM&page=1">Try Ubuntu confidential VMs today&nbsp;&rsaquo;</a>
```

Comment:
Please wrap the anchor tag in a `p`, this was a comment that I received from MPT from a previous PR